editor/src/components/editor/actions/actions.ts

```diff
@@ -1517,6 +1519,18 @@ function updateSelectedComponentsFromEditorPosition(
   }
 }
 
+function removeModulesFromNodeModules(
+  modulesToRemove: Array<string>,
+  nodeModules: NodeModules,
+): NodeModules {
+  const filePathsToRemove = modulesToRemove.map((m) => `/node_modules/${m}/`)
+  return Object.fromEntries(
```

Comment:
I've been using the badly named objectMapDropNulls for this https://github.com/concrete-utopia/utopia/blob/6fba04e8ab53daefdba16ad4aa7e0e392d89b804/editor/src/core/shared/object-utils.ts#L126

Reply:
Ah good shout, that would make this a lot clearer